pages/marketplace/orders/index.tsx

```diff
@@ -25,12 +25,21 @@ import {
 import { Content, SubtitleCentered } from '../../../styles/marketplace/styles'
 
 const MyOrders: NextPage = () => {
-  const { account, reconnect } = useContext(Web3Context)
-  const { orders } = useContext(MarketplaceContext)
-  const myOrders = orders.filter((o) => o.seller.toString() === account)
+  const [myOrders, setMyOrders] = useState([])
+  const axiosHttpClient = new AxiosHttpClient()
+  const remoteLoadOrders = new RemoteLoadOrders(
+    `https://dev-nft.api.mousehaunt.com/marketplace/orders/search?mouse_box=true&box_types=["heroic","genesis"]&from_price=100000&to_price=10000000&from_amount=1&to_amount=1&offset=0&limit=10`,
+    axiosHttpClient
+  )
```

Comment:
Como conversamos na call, colocar em uma estrutura para puxar da `config[network]`
```suggestion
  const remoteLoadOrders = new RemoteLoadOrders(
    `${config[network].api}/marketplace/orders/search?mouse_box=true&box_types=["heroic","genesis"]&from_price=100000&to_price=10000000&from_amount=1&to_amount=1&offset=0&limit=10`,
    axiosHttpClient
  )
```
Acredito que os parâmetros das chamadas já podem ser setados numa lógica que sejam variáveis : 

```suggestion
const fromPrice = '100000'
const toPrice = '10000000'

const params = `from_price=${fromPrice}` 
                           + `&to_price=${toPrice}` // (Fazer para os demais)
  const remoteLoadOrders = new RemoteLoadOrders(
    `${config[network].api}/marketplace/orders/search?${params}`,
    axiosHttpClient
  )
```